HSEARCH-5464 Pluggable rest clients in the elasticsearch backend (#4798)
hibernate/hibernate-search: documentation/src/main/asciidoc/public/reference/_backend-elasticsearch.adoc

== Client configuration

An Elasticsearch backend communicates with an Elasticsearch cluster through a REST client.

> Hibernate Search comes with a range of pluggable Elasticsearch REST clients that the user can pick from:

Reviewer: "User" means something different for us and for... our users.

```suggestion
Hibernate Search comes with a range of pluggable Elasticsearch REST clients that you can pick from:
```

or

```suggestion
Hibernate Search comes with a range of pluggable Elasticsearch REST clients that applications can pick from:
```
